src/ctapipe/image/muon/tests/test_muon_features.py

```diff
@@ -50,3 +55,70 @@ def test_ring_completeness():
 
     assert ring_comp <= 1
     assert ring_comp >= 0
+
+
+def test_ring_size_parameters():
+    radius = 1
+    center_x = 0
+    center_y = 0
+    pixel_x = np.linspace(-2, 2, 1855)
+    pixel_y = np.linspace(-2, 2, 1855)
+    ring_integration_width = 0.25
+    outer_ring_width = 0.2
+    image = np.random.normal(loc=100, scale=10, size=1855)
+    image_mask = np.random.choice([True, False], size=1855)
+
+    (
+        ring_size,
+        size_outside,
+        num_pixels_in_ring,
+        mean_pixel_outside_ring,
+    ) = ring_size_parameters(
+        radius,
+        center_x,
+        center_y,
+        pixel_x,
+        pixel_y,
+        ring_integration_width,
+        outer_ring_width,
+        image,
+        image_mask,
+    )
+
+    assert ring_size > 0
```

Comment:
I think this test is not definitive enough. Please consider writing a test actually being sensitive on changes other than the value being <= 0.

I.e. use the ring gaussian toymodel to create a ring with known properties:
https://github.com/cta-observatory/ctapipe/blob/7858ddc3575681d59a266ecc1f20d1510f2f3555/src/ctapipe/image/toymodel.py#L375